openslides_backend/action/actions/projector/control_view.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Dict
+
+from ....models.models import Projector
+from ....shared.patterns import FullQualifiedId
+from ...generics.update import UpdateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+
+
+@register_action("projector.control_view")
+class ProjectorControlView(UpdateAction):
+    """
+    Action to control view a projector.
+    """
+
+    model = Projector()
+    schema = DefaultSchema(Projector()).get_update_schema(
+        additional_required_fields={
+            "field": {"type": "string", "enum": ["scale", "scroll"]},
+            "direction": {"type": "string", "enum": ["up", "down", "reset"]},
+        },
+        additional_optional_fields={
+            "step": {"type": "integer"},
+        },
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        field = instance.pop("field")
+        direction = instance.pop("direction")
+        step = instance.pop("step", 1)
```

Comment:
I missed one thing in the docs: if `step < 1`, step is set to 1.

Reply:
I catch the `step < 1` and add a test for it. A step < 1 is an error.